homeassistant/components/switchbot/switch.py

```diff
@@ -77,3 +84,10 @@ def unique_id(self) -> str:
     def name(self) -> str:
         """Return the name of the switch."""
         return self._name
+
+    @property
+    def device_state_attributes(self):
```

Comment:
Could you also` add -> Dict[str, Boolean]`

Reply:
Done